### PR TITLE
Updated files

### PR DESCRIPTION
**Pull request by TetianaHuk**

### CODE REVIEW

**Comment by harbuzniak** [comment]
@TetianaHuk my.json and lo.json, please remove extra symbols in the beginning of files

File: frontend/lo.json [hunk 1]
```diff
@@ -1,69 +1,69 @@
-{
-    "search_button": "ຄົ້ນຫາໃໝ່",
-    "review_question": "ພວກເຮົາດີທີ່ເຮົາໃຫ້ທ່ານ?",
-    "leave_review": "ການປະມູນວຽກຄືນ",
-    "review": "ພວກເຮົາ",
+x{
```
Comment (harbuzniak) [inline frontend/lo.json]: @TetianaHuk remove extra symbol

File: frontend/my.json [hunk 2]
```diff
@@ -1,69 +1,69 @@
-{
-    "search_button": "ကျွန်ုပ်တို့ရှာဖွေလိုက်ပါ",
-    "review_question": "ကျွန်ုပ်အောင်မြင်လား?",
-    "leave_review": "ကျွန်ုပ် ပြောင်းရန်",
-    "review": "ကျွန်ုပ်အောင်မြင်ခြင်း",
+[{
```
Comment (harbuzniak) [inline frontend/my.json]: @TetianaHuk remove extra symbol

**Review by harbuzniak — CHANGES_REQUESTED** [review]

File: backend/messages.sm.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
-public_tracking_page:
-    search:
-        email_response: "Mo fesiligia mo so'o se tulafono, o lo outou i'ugafono auina atu le fa'amaotiina o se lagolago aupito (ina ia maua ai).<br /> Fa'amolemole fa'ao'auga i lou i'ugafono e va'ai fa'atasi ai le mea na totoe ai lau fa'amatalaga."
+public_tracking_page:
+    search:
+        email_response: "Mo mafuaaga saogalemu ua matou lafoina lau fesoʻotaʻiga i lau imeli ( pe a iai ) . < br / > Faʻamolemole siaki lau imeli e vaʻai poʻo fea o iai lau fanua.
```
Comment (harbuzniak) [inline backend/messages.sm.yml]: @TetianaHuk missed " at the end of message

**Review by harbuzniak — CHANGES_REQUESTED** [review]

**Comment by TetianaHuk** [comment]
@harbuzniak Fixes were added, please, revise)

File: languages.json [hunk 4]
```diff
@@ -1,517 +1,517 @@
...
+  },
+  {
+    "name": "Sinhala",
+    "code": "si",
+    "active": "yes"
+  },
+  {
+    "name": "Turkmen",
+    "code": "tk",
+    "active": "yes""
```
Comment (harbuzniak) [inline languages.json]: @TetianaHuk remove extra "

**Review by harbuzniak — CHANGES_REQUESTED** [review]

File: languages.json (hunk 4 above)
Comment (TetianaHuk) [reply languages.json]: Extra sign was removed, please revise)

**Pull request merged**